lib/spack/spack/cmd/env.py

```diff
@@ -272,6 +278,39 @@ def env_list(args):
     colify(color_names, indent=4)
 
 
+#
+# env view
+#
+def env_view_setup_parser(subparser):
+    """manage a view associated with the environment"""
+    views = subparser.add_mutually_exclusive_group()
+    views.add_argument(
+        '--regenerate', action='store_true',
+        help="regenerate the view for the environment")
+    views.add_argument(
+        '--enable', action='store_true',
+        help="enable a view for the environment")
+    views.add_argument(
+        '--disable', action='store_true',
```

Comment:
I'm not sure which commands you mean. But perhaps more importantly, would you rather the command syntax was something like:

```
spack env view-disable
spack env view-enable
spack env view-regenerate
```

vs. 

```
spack env view --disable
spack env view --enable
spack env view --regenerate
```

? I'm inclined toward the latter since the former could get unwieldy.

Reply:
I think it's already the latter, and @tgamblin is requesting

```
spack env view disable
spack env view enable
spack env view regenerate
```

Whether or not I'm right that it's what Todd was suggesting, I think that would be an improvement on what I wrote and fit best with the usage in the rest of Spack.

Reply:
Yep, what @becker33 said -- `spack env view disable` etc.

Reply:
There is now a view action argument which can be one of `enable/disable/regenerate`. I felt the closest analogy was `spack config edit <section>`.